src/components/routes/homepage/SearchBar.js

```diff
@@ -57,8 +59,11 @@ export const SearchBar = ({ setFilter, sortParam, setSortParam }) => {
           value={sortParam}
           onChange={({ target: { value } }) => setSortParam(value)}
         >
-          <MenuItem value={'Date'}>Date</MenuItem>
-          <MenuItem value={'Subscriber'}>Subscriber</MenuItem>
+          {sortParams.map((elem, index) => (
```

Comment:
should not be lazy lol... use `param` instead of `elem`

Reply:
😂 will make a quick commit on this